docs/releasenotes.md

```diff
@@ -5,6 +5,19 @@ title: Releasenotes
 
 # Release notes BAG Huidige bevragingen API
 
+## Versie 1.3.0 (20-09-2021)
+In deze release is een bugfix doorgevoerd ten aanzien van mogelijk onjuist bij een adres. Tevens zijn er in het kader van de opgeloste issues een feature-file toegevoegd en enkele wijzigingen aan de API-specificaties doorgeveord.
```

Comment:
In deze release is een bugfix opgenomen over "mogelijk onjuist" bij een adres. Naast wijzigingen in de API specificaties  is een feature-file toegevoegd.

Reply:
Aangepast